aries-site/src/examples/foundation/color/ColorSwatches.js

```diff
@@ -0,0 +1,288 @@
+    return num(a.id) - num(b.id);
+  });
+
+  // Split into two columns for top-to-bottom, left-to-right flow
+  const midpoint = Math.ceil(sortedTokens.length / 2);
+  const left = sortedTokens.slice(0, midpoint);
+  const right = sortedTokens.slice(midpoint);
+
+  return (
+    <SwatchGroup>
+      <Box direction="column" gap="medium">
+        {left.map(t => (
+          <ColorSwatch key={t.id} background={t.value} text={getLabel(t)} />
+        ))}
+      </Box>
+      <Box direction="column" gap="medium">
+        {right.map(t => (
+          <ColorSwatch key={t.id} background={t.value} text={getLabel(t)} />
+        ))}
+      </Box>
+    </SwatchGroup>
+  );
+};
+
+const TextColorSwatch = ({ color, text }) => (
+  <Box direction="row" gap="small" align="center">
+    <Text size="xlarge" weight="bold" color={color}>
+      Aa
+    </Text>
+    <Text>{text}</Text>
+  </Box>
+);
+TextColorSwatch.propTypes = {
+  color: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+};
+
+export const TextSwatch = () => {
+  const textTokens = filterByPrefix(useColorTokens(), 'hpe.color.text.');
+  const sampleTextTokens = pickTokens(textTokens, [
+    'hpe.color.text.default',
+    'hpe.color.text.strong',
+    'hpe.color.text.weak',
+  ]);
+
+  return (
+    <TokenSwatchList tokens={sampleTextTokens} component={TextColorSwatch} />
+  );
+};
+
+const IconColorSwatch = ({ color, text }) => (
+  <Box direction="row" gap="small" align="center">
+    <Add color={color} size="medium" />
+    <Text>{text}</Text>
+  </Box>
+);
+IconColorSwatch.propTypes = {
+  color: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+};
+
+export const IconSwatch = () => {
+  const iconTokens = filterByPrefix(useColorTokens(), 'hpe.color.icon.');
+  const sampleIconTokens = pickTokens(iconTokens, [
+    'hpe.color.icon.default',
+    'hpe.color.icon.strong',
+    'hpe.color.icon.weak',
+  ]);
+
+  return (
+    <TokenSwatchList tokens={sampleIconTokens} component={IconColorSwatch} />
+  );
+};
+
+const StatusColorSwatch = ({
+  iconColor,
+  backgroundColor,
+  iconText,
+  backgroundText,
+}) => (
+  <Box direction="row" gap="small" align="center">
+    <Box
+      background={backgroundColor}
+      align="center"
+      justify="center"
+      round="small"
+      pad="5xsmall"
+      width="5xsmall"
+      height="5xsmall"
+    >
+      <Box background={iconColor} round="full" width="24px" height="24px" />
+    </Box>
+    <Box>
+      <Text>{iconText}</Text>
+      <Text>{backgroundText}</Text>
+    </Box>
+  </Box>
+);
+StatusColorSwatch.propTypes = {
+  iconColor: PropTypes.string.isRequired,
+  backgroundColor: PropTypes.string.isRequired,
+  iconText: PropTypes.string.isRequired,
+  backgroundText: PropTypes.string.isRequired,
+};
+
+export const StatusSwatch = () => {
+  const colorTokens = useColorTokens();
+
+  const statusTypes = ['ok', 'warning', 'critical', 'info', 'unknown'];
```

Comment:
@sulaymon-tajudeen-hpe - Add comment for why these needed to be specified
```suggestion
  // Status colors are assembled across a variety of tokens which do not allow for easy programmatic selection from their namespace. @oliverHPE suggested a possible solution is to introduce a token $subtype in the future.
  const statusTypes = ['ok', 'warning', 'critical', 'info', 'unknown'];
```